index.html

```diff
@@ -221,9 +221,19 @@
         <ul id="segment-metadata" class="col-8"></ul>
       </div>
     </div>
-
   </div>
 
+  <div class="text-center p-3" id=unit-test-link>
+    <a href="test/debug.html">Run unit tests</a>
+  </footer>
+
+  <script>
+    var unitTestLink = document.getElementById('unit-test-link');
+
+    if ((/netlify.app/).test(window.location.href.host)) {
+      unitTestLink.parentElement.removeChild(unitTestLink);
```

Comment:
since we don't care about IE in this file, we can also use [`remove()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove)
```suggestion
      unitTestLink.remove();
```

Reply:
fancy